auth-api/src/auth_api/models/affiliation_invitation.py

```diff
@@ -50,6 +50,7 @@ class AffiliationInvitation(BaseModel):  # pylint: disable=too-many-instance-att
     invitation_status_code = Column(ForeignKey('invitation_statuses.code'), nullable=False, default='PENDING')
     type = Column(ForeignKey('affiliation_invitation_types.code'), nullable=False, default='EMAIL')
     additional_message = Column(String(4000), nullable=True)
+    is_deleted = Column(Boolean(), default=False)
```

Comment:
might make more sense to put this as `is_viewed`?  `is_deleted` is fine though

Reply:
Will keep is as `is_deleted` then, thanks